test/test_matmul_scheduler.cpp

```diff
@@ -15,16 +15,971 @@
 #include <scheduler/mma_utils.h>
 #include <test/utils.h>
 #include <test/validator.h>
+#include "ops/arith.h"
+#include "type.h"
 
 namespace nvfuser {
 
 namespace {
 class MatmulSchedulerTest : public NVFuserTest {};
+
+using PrecisionsDesc = std::pair<PrimDataType, PrimDataType>;
+
+using AbsoluteError = double;
+using RelariveError = double;
+using ErrorThresholds = std::pair<AbsoluteError, RelariveError>;
+using TestCaseErrorThresholds = std::map<PrecisionsDesc, ErrorThresholds>;
+class PrecisionParametrizedTest
+    : public NVFuserFixtureParamTest<PrecisionsDesc> {};
+
+[[nodiscard]] auto getAtType(const PrimDataType& type) {
```

Comment:
Should we use https://github.com/NVIDIA/Fuser/blob/52b98c77cfe312ab7a60171f4befd292c42c234f/csrc/type.h#L872 ?

Reply:
I wasn't aware of this helper - will use it. Thanks!